lib/telegraph/client.rb

```diff
@@ -0,0 +1,67 @@
+module Telegraph
+  class Client
+    include Telegraph::Connection
+
+    attr_reader :token
+
+    def initialize(token)
+      @token = token
+    end
+
+    def create_account(attrs = {})
+      acc_info = get('createAccount', attrs)
+      @account = Account.new(acc_info)
+    end
+
+    def account
+      @account ||= get_account_info
+    end
+
+    def get_account_info(*fields)
+      # Gets all available account information if 'fields' array not given.
+      available_fields = %w(short_name author_name author_url auth_url page_count)
```

Comment:
a) constant
b) `available_` says nothing, `DEFAULT_FIELDS` would be probably better for constant name